File: C/1.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
	int t;
	string str;
	float avg;
	string max_name="";
	float max_avg = -1;
	
	cin>>t;
	
	for(int i = 0; i<t; i++)
	{
		cin >> str >> avg;
		if(avg > max_avg)
		{
			max_avg = avg;
			max_name = str;
		}
	}
	cout<<max_name;	
}

File: python/1.py
#!/usr/bin/env python
# coding: utf-8

# In[ ]:


max_name=""
max_avg = -1
t = int(input())
for i in range(t):
    name_avg = list(input().split())
    name = name_avg[0]
    avg = float(name_avg[1])
    if avg > max_avg:
        max_avg = avg
        max_name = name
print(max_name)

